### DISCUSSION TITLE
PCB Manufacturing

### DISCUSSION THREAD

**Discussion by unintended-consequences** [revk/xESP32-Faikout#29] [General]
Looks like A&A is no longer making the PCB, price has been removed from their website and also it's marked unavailable on Amazon.co.uk.

Can I just take the gerber.zip to any local fabricator and get a PCB made? (apols, this is my first time making a PCB)

**Comment by unintended-consequences** [comment]
Ahh. I guess I need a parts list too to solder to the board? ESP32 and also legs? I guess ordering from China pre-soldered might be better if there's a suitable vendor?

**Reply by unintended-consequences** [reply]
Oh, that's the bom.csv! Bill of Materials. Seems more complicated than I thought. Will research completed PCB manufacturing. But if you have any suggestions, would love some hints!

**Comment by revk** [comment]
It is on Amazon if you have a UK address, they are refusing to export for some reason. Some customers found a forwarding company that works forward2me.com

**Comment by unintended-consequences** [comment]
Thanks. I've ordered via VPN. Now I need to figure out how to source a cable...

**Reply by JeanOlivier** [reply]
eBay, got mine that way.

**Comment by JeanOlivier** [comment]
For the record, I was able to order and ship to Canada through Amazon.co.uk with no complications a few days ago and I juste received the board.

I'll test over the weekend to see if it'll support my North American unit!

Edit:

And it does work!

I just needed to do the OTA update!
Thanks a million!

I'll edit this later with the exact model so people looking for it can know it works.

Edit:

My model is CTXS09HVJU.

**Comment by revk** [comment]
Current listing is https://www.amazon.co.uk/dp/B0C2ZYXNYQ
